### PR TITLE
Support HGVSg and Genomic Change annotation

### PR DESCRIPTION
**Pull request by zhx828**
This fixes https://github.com/oncokb/oncokb-annotator/issues/69 and https://github.com/oncokb/oncokb-annotator/issues/70

### CODE REVIEW

File: AnnotatorCore.py [hunk 1]
```diff
@@ -281,113 +309,227 @@ def processalterationevents(eventfile, outfile, previousoutfile, defaultCancerTy
...
+        return ''
+    else:
+        return None
 
-            if len(queries) == POST_QUERIES_THRESHOLD:
-                annotations = pull_mutation_info(queries)
-                append_annotation_to_file(outf, rows, annotations)
-                queries = []
-                rows = []
+def process_alteration(maffilereader, outf, maf_headers, alteration_column_names, ncols, defaultCancerType, cancerTypeMap,
```
Comment (zhx828) [inline AnnotatorCore.py:338]: No need to review this method, it's the same code from top left.

File: MafAnnotator.py [hunk 2]
```diff
@@ -66,6 +77,7 @@ def main(argv):
     parser.add_argument('-a', dest='annotate_hotspots', action="store_true", default=False)
     parser.add_argument('-v', dest='cancer_hotspots_base_url', default='', type=str)
     parser.add_argument('-b', dest='oncokb_api_bearer_token', default='', type=str)
+    parser.add_argument('-q', dest='alteration_query_type', default=AlterationQueryType.HGVSP_SHORT.value, type=str)
```
[the remark this answers is not in the document]
Comment (jjgao) [reply MafAnnotator.py:91]: Maybe just QueryType? I don't feel strongly about it.

[the remark this answers is not in the document]
Comment (jjgao) [reply MafAnnotator.py:91]: For the default, is it possible for the script to automatically select based on an order? e.g. hgvsp_short, hgvsp, hgvsg, genomic_change?

File: example.sh [hunk 3]
```diff
@@ -12,8 +16,15 @@ OC="data/example_clinical.oncokb.txt"
 OCPDF="data/example_clinical.oncokb.pdf"
 TOKEN="" #OncoKB API Token
 README="data/example_README.txt"
+
 python MafAnnotator.py -i $IMAF -o $OMAF -c $IC -b $TOKEN
+python MafAnnotator.py -i $IMAF -o $OMAFHGVSPSHORT -c $IC -b $TOKEN -q hgvsp_short
+python MafAnnotator.py -i $IMAF -o $OMAFHGVSP -c $IC -b $TOKEN -q hgvsp
+python MafAnnotator.py -i $IMAF -o $OMAFHGVSG -c $IC -b $TOKEN -q hgvsg
+python MafAnnotator.py -i $IMAF -o $OMAFGC -c $IC -b $TOKEN -q genomic_change
```
[the remark this answers is not in the document]
Comment (jjgao) [reply example.sh:24]: Good idea! But maybe we just need to use the same input MAF and generate different output MAF?

[the remark this answers is not in the document]
Comment (zhx828) [reply example.sh:24]: This is the case if you look at the -i parameter.

**Review by jjgao — APPROVED** [review]

**Pull request merged**